src/components/test/SvgIcon-test.js

```diff
@@ -12,9 +17,13 @@ describe('SvgIcon', () => {
   // Global icon set that is registered with `SvgIcon` outside of these tests.
   let savedIconSet;
 
-  beforeEach(() => {
-    savedIconSet = availableIcons();
+  before(() => {
+    // Clone the returned Map so that we're not affected by changes to the
+    // Map it references
+    savedIconSet = new Map(availableIcons());
```

Comment:
Ran into some "interesting" behavior here during my travels.

Reply:
`availableIcons` is only used in tests, and is thus not performance sensitive, so we could make it return a map copy itself.